lib/msf/core/exploit/local/windows_kernel.rb

```diff
@@ -81,7 +81,7 @@ def find_sys_base(drvname)
       print_error("EnumDeviceDrivers failed (error: #{results['GetLastError']} #{results['ErrorMessage']})")
       return nil
     end
-    addresses = results['lpImageBase'][0..results['lpcbNeeded'] - 1].unpack("#{ptr}*")
+    addresses = results['lpImageBase'][0..results['lpcbNeeded'] - 1].unpack("#{ptr}")
```

Comment:
This looks like a wrong change 👀

Reply:
Ah, good catch. You're spot on, it's from before the string interpolation handling was removed. Will get that reverted.